docs/_toc.yml

```
@@ -98,6 +98,10 @@ subtrees:
         subtrees:
         - entries:
           - file: developers/contributing
+          - file: naps/index
+            subtrees:
+            - entries:
+              - file: naps/0-nap-process
```

Here's one possible solution:
```suggestion
          - file: naps/index
            subtrees:
            - maxdepth: 1
              entries:
              - file: naps/0-nap-process
```

With this change, instead of doing 

````


```{toctree}
---
maxdepth: 1
---
0-nap-process
```

````

in the index.md file, I would use 

````

```{tableofcontents}
```

````